torch/_inductor/dependencies.py

```diff
@@ -696,9 +696,11 @@ def extract_input_node_reduction_ranges(
 
             if isinstance(op, ComputedBuffer) and len(op.get_reduction_size()) > 0:
                 if reduction_size is None:
-                    reduction_size = op.get_reduction_size()
-                    size = op.get_size()
-                elif reduction_size != op.get_reduction_size() or size != op.get_size():
+                    reduction_size = [*op.get_reduction_size()]
+                    size = [*op.get_size()]
+                elif reduction_size != [*op.get_reduction_size()] or size != [
```

Comment:
Why convert the returned sequence into a list? Optional[List[T]] should be comparable to Sequence[T]

Reply:
``
>>> (1, ) == [1, ]
False
>>> 
```